### PR TITLE
Circle ci

### PR DESCRIPTION
**Pull request by pyaillet**

### CODE REVIEW

**Comment by pyaillet** [comment]
@bgauduch this one should be ready for review. Some more tests with tags might be useful though before shutting down dockerhub automated builds. WDYT ?

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      - lint
+      - build:
+          requires:
+            - lint
+      - build-tag:
+          requires:
+            - lint
+          filters:
+            tags:
+              only: /v?[0-9]+\.[0-9]+/
```
Comment (bgauduch) [inline .circleci/config.yml]: I think we can get rid of the v in the tag, to folow the current project tag convention 😉

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            branches:
+              ignore: /.*/
+      - publish:
+          requires:
+            - build
+            - build-tag
+          filters:
+            tags:
+              only: /v?[0-9]+\.[0-9]+/
```
Comment (bgauduch) [inline .circleci/config.yml]: I think we can get rid of the v in the tag, to folow the current project tag convention 😉

**Comment by bgauduch** [comment]
Yes we can do a tag test once it's merged in master 😉

File: debian.Dockerfile [hunk 3]
```diff
@@ -6,23 +6,26 @@ ARG TERRAFORM_VERSION=0.12.9
 FROM debian:stretch-20190506-slim as terraform
 ARG TERRAFORM_VERSION
 RUN apt-get update
+# hadolint ignore=DL3015
 RUN apt-get install -y curl=7.52.1-5+deb9u9
-RUN apt-get install -y unzip=6.0-21+deb9u2
-RUN apt-get install -y gnupg=2.1.18-8~deb9u4
+RUN apt-get install -y unzip=6.0-21+deb9u2 --no-install-recommends
+RUN apt-get install -y gnupg=2.1.18-8~deb9u4 --no-install-recommends
```
Comment (bgauduch) [inline debian.Dockerfile]: Well done for the CI, although it's only in stage 👍

**Comment by pyaillet** [comment]
And "PAF" as they say in France

**Pull request merged**